### PR TITLE
PaymentServiceClient up and running, passing unit tests

### PR DESCRIPTION
**Pull request by callum-4**
Screenshot 2024-11-12 190207

### CODE REVIEW

File: game-service/.env
Comment (Whomstt) [inline game-service/.env:1]: whoops env

File: information-service/src/main/resources/application.properties
Comment (Whomstt) [inline information-service/src/main/resources/application.properties:1]: We must do different ports for each service for local hosting. I have notifications on 8085 now. good shtuff

**Review by Whomstt — APPROVED** [review]
Looks good to me just the note with the env file. Also side note to add your notification service to the root pom.xml for easy mvn install. Good work.

Update: unit tests are failing

**Comment by olanhealy** [comment]
Story with this?

**Pull request closed without merging**